packages/widget-chat/src/reducers/conversation.js

```diff
@@ -30,7 +38,25 @@ export default function conversation(state = {
     });
   }
 
-  default:
+  case RECEIVE_MERCURY_ACTIVITY: {
```

Comment:
It's in the action file where we call sdk. Wasn't sure if that logic should be there or here. Initial thought is that filtering should happen before the action actually gets dispatched so we don't dispatch with data that doesn't actually change state. Thoughts?